05-script.md

```diff
@@ -440,3 +440,34 @@ Of course, this introduces another tradeoff between flexibility and complexity.
 > # Script 3
 > echo $@.dat
 > ~~~
+
+> ## Ooops my large script has a bug. How can I easily find the problem? {.challenge}
+>
+> You have written a large script with many lines, and it's not working properly.
+> How can I easily identify the problem ?
```

Comment:
> Interpreting the 'output' might not be 'easy' but at least someone can look at it, hopefully spot the error and they have something to provide them with an idea as to where they went wrong.

Sorry if this came across wrong -- I do not argue with the usefulness of `bash -x`, it is certainly easier to debug the script with it! What I meant to say was that by saying "How can you easily identify the problem" you are implying that with `bash -x` the learner will "easily identify the problem". It might be frustrating for the learner if that does not quite work for them after they were told it was "easy".
Not a major point and others might disagree with my view here, so don't worry about it too much ;)